### PR TITLE
#117 more tests

### PR DESCRIPTION
**Pull request by stereobooster**

### CODE REVIEW

File: tests/examples/css-variables.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
```
Comment (stereobooster) [inline tests/examples/css-variables.css]: this is what they use in bootstrap

File: tests/main.test.js [hunk 2]
```diff
@@ -152,3 +152,23 @@ test("deliberately skipped .css shouldn't error", async () => {
...
 })
+
+test.skip('leaves used css variables', async () => {
+  const { finalCss } = await runMinimalcss('css-variables')
+  expect(finalCss).toMatch('--main-bg-color:')
+})
+
+test.skip('removes unused css variables', async () => {
+  const { finalCss } = await runMinimalcss('css-variables')
+  expect(finalCss).not.toMatch('--unused-color:')
```
Comment (peterbe) [inline tests/main.test.js]: I think these two above tests test almost the same thing and thus they can be in the same test. 
E.g.
```javascript
test.skip('css variables', async () => {
  expect.assertions(2)
  const { finalCss } = await runMinimalcss('css-variables')
  expect(finalCss).toMatch('--main-bg-color:')
  expect(finalCss).not.toMatch('--unused-color:')
})

File: tests/main.test.js [hunk 3]
```diff
@@ -152,3 +152,23 @@ test("deliberately skipped .css shouldn't error", async () => {
   })
   expect(finalCss).toEqual('p{color:brown}')
 })
+
+test.skip('leaves used css variables', async () => {
```
Comment (peterbe) [inline tests/main.test.js]: Can you add a comment above this one *why* this test is currently skipped. 
Perhaps a comment that contains a URL to a filed issue about it.

File: tests/examples/css-variables.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link href="css-variables.css" rel=stylesheet>
+  </head>
+  <body>
+    <p>Only a P tag in this document. And images too.</p>
```
Comment (peterbe) [inline tests/examples/css-variables.html]: There are no images in this HTML.

File: tests/examples/css-variables.css (hunk 1 above)
Comment (peterbe) [reply tests/examples/css-variables.css]: Perhaps that should be in a CSS comment right before. So we don't forget.

**Comment by peterbe** [comment]
@stereobooster wanna rebase and land this?

**Comment by peterbe** [comment]
@stereobooster Feel free to squash and merge.

**Pull request merged**